### DISCUSSION TITLE
Duplicate Electrical Bus in LMP by Electrical Bus reports

### DISCUSSION THREAD

**Discussion by bryn-gi** [ercot/api-specs#53] [Q&A]
I noticed there are multiple duplicate electrical bus rows in the LMP by Electrical Bus report, just wondering why that is the case? Why are there duplicates?

**Comment by CaffeineLab** [comment]
This won't be the best answer, but, if you look in the list of [settlement points](https://www.ercot.com/mp/data-products/data-product-details?id=NP4-160-SG) you'll see how it is broken down:
![image](https://github.com/user-attachments/assets/d09a09bc-b25a-422c-9b29-fba1e78f9b23)

and in the lmp price by bus it's broken out .. but you are missing that one thing to tie it together:
![image](https://github.com/user-attachments/assets/e4dff9ca-7a9f-4a28-94b2-e9631dbf77e6)
